Makefile

```diff
@@ -36,7 +36,6 @@ build-all: proto-lint proto-gen update-swagger-docs build
 ### Analyzing
 
 lint:
-	GO111MODULE=off go get github.com/golangci/golangci-lint/cmd/golangci-lint@v1.30.0
 	golangci-lint run --timeout 5m0s --allow-parallel-runners
```

Comment:
Could you comment why we need to remove this statement? This statement was for installing the golintci-lint binary.

Reply:
Oh, I'll roll this back.
It seems to have been deleted by mistake.

Reply:
I fixed it. (https://github.com/medibloc/panacea-core/pull/375/commits/182ace030e11cbb3e463edd8702e01b310463216)